### PR TITLE
Added the SR feedback announcement and the describedby region for stu…

### PR DESCRIPTION
**Pull request by staubina**
…dent training

### Description

Rubric questions are now aria described by feedback sections when they are right above the text, like in Student training. Per a conversation with @cptvitamin this only pertains to the sections of feedback in the student training.

### Sandbox
- [x] Build a sandbox for your branch and add a link here

https://staubina-ora3.sandbox.edx.org/courses/course-v1:edx+ORA203+course/courseware/676026889c884ac1827688750871c825/5663e9b038434636977a4226d668fe02/

### Testing
- [x] Unit, integration, acceptance tests as appropriate

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @cahrens 
- [x] Code review: @efischer19 
- [x] Accessibility review: @cptvitamin 

FYI:
@jlajoie 
### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

File: openassessment/templates/openassessmentblock/student_training/student_training.html [hunk 1]
```diff
@@ -89,41 +89,39 @@ <h5 class="message__title">{% trans "Learning to Assess Responses" %}</h5>
...
-                                                               value="{{ option.name }}"
-                                                               name="training__assessment__rubric__question--{{ criterion.order_num }}"/>
-                                                        <label for="training__assessment__rubric__question--{{ criterion.order_num }}__{{ option.order_num }}__{{ xblock_id }}"
-                                                               class="answer__label">{{ option.label }}</label>
-                                                    </div>
-                                                    <div class="wrapper--metadata">
-                                                        <span class="answer__tip">{{ option.explanation }}</span>
-                                                        <span class="answer__points">{{option.points}} <span class="answer__points__label">{% trans "points" %}</span></span>
-                                                    </div>
+                                        <div class="question__answers" role="group" aria-labelledby="training__assessment__rubric__prompt--{{ criterion.order_num }}__{{ xblock_id }}">
```
Comment (staubina) [inline openassessment/templates/openassessmentblock/student_training/student_training.html]: Note: I merged two divs and updated the indentation.

**Comment by cptvitamin** [comment]
@staubina I see the ids added to the feedback containers, but I don't see them referenced via aria-describedby on the group container.  I might be missing it though. I didn't see it in the sandbox either after making selections in Step 2.

**Comment by cptvitamin** [comment]
👍 

NOTE: @staubina clarified in person and I confirmed everything is working as expected

**Comment by staubina** [comment]
@cahrens @efischer19 Updates made.  I am waiting on the base PR to merge.

**Pull request merged**